generator.go

```diff
@@ -30,6 +30,8 @@ var (
 	// MigrationTemplate for the migration generator.
 	//go:embed templates/migration.xml.tmpl
 	migrationTemplate string
+	//go:embed templates/changelog.xml.tmpl
+	changelogTemplate string
```

Comment:
Love it!

Reply:
I didn't know this trick existed, but catched the idea :)